### PR TITLE
Remove debug statements

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@ Change history
 2.0.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Remove debug statements checker, flake8 plugin and jshint can do the
```
Comment (tisto) [inline CHANGES.rst]: Following semantic versioning, the next version should be 2.1 then.

**Comment by gforcada** [comment]
@tisto good point, updated the next release version to 2.1

**Comment by do3cc** [comment]
I miss the information in the docs, which flake8 plugins provide this feature.

**Comment by gforcada** [comment]
@do3cc you are right, I did that branch on the tram and I had no Internet connection, I was planning to add it afterwards, thanks for the reminder!

**Comment by gforcada** [comment]
I even found a bit misleading the name as I expected that pdb usage would also get a warning, but we were only checking for print statements.

I pointed to both of them on the changelog entry.

**Comment by hvelarde** [comment]
occurrences of `pdb` are being tested here: https://github.com/plone/plone.recipe.codeanalysis/blob/master/plone/recipe/codeanalysis/tests/test_flake8.py#L63:L72

BTW, seem that the tests for ipdb have a typo; I fixed here: #159

**Comment by tisto** [comment]
@gforcada did you update the docs/changelog yet? Apart from this I think this pr can be merged.

**Comment by hvelarde** [comment]
squash all commits into one first, please

**Comment by gforcada** [comment]
@tisto yes I added a changelog entry and bumped the version to 2.1, travis was complaining let's see if it's ok now...

**Comment by tisto** [comment]
@gforcada can we add the new flake8-plugins to our README as "best practice"?

**Comment by gforcada** [comment]
I was thinking that we should copy zest.releaser and create a ``[recommended]`` extra for p.r.codeanalysis where we add all these flake8 plugins that we think should be... ehem .. recommended, does it make sense?

As you can see from my last tweets I'm basically stripping everything python-only related from p.r.codeanalysis and putting it on its own distribution, so we can make them more flexible and useful for others and have different development cycles, while on p.r.codeanalysis should be basically the syntactic sugar to put everything together.

**Comment by tisto** [comment]
[recommended] sounds good to me. Though, I would also be ok with just adding our recommended plugins to the README. I usually copy the config from there anyways.

Regarding your flake8-plugins: That's really awesome! I always had that vision that p.r.codeanlysis just becomes a thin layer around generic python code analysis tools. Thanks a lot for working on this!!! Factor out as much as possible, less options is good.

**Pull request merged**